src/Generator/TaxItemsGeneratorInterface.php

```diff
@@ -4,15 +4,16 @@
 
 namespace Sylius\RefundPlugin\Generator;
 
+use Doctrine\Common\Collections\Collection;
+use Sylius\RefundPlugin\Entity\LineItemInterface;
 use Sylius\RefundPlugin\Entity\TaxItemInterface;
-use Sylius\RefundPlugin\Model\UnitRefundInterface;
 
 interface TaxItemsGeneratorInterface
 {
     /**
-     * @param array<UnitRefundInterface> $units
+     * @param Collection|LineItemInterface[] $lineItems
      *
      * @return array<TaxItemInterface>
      */
-    public function generate(array $units): array;
+    public function generate(Collection $lineItems): array;
```

Comment:
Is there any reason to accept a `Collection` instead of an array?

Reply:
The same as with https://github.com/Sylius/RefundPlugin/pull/179#discussion_r372227908